js/src/views/Account/Transactions/transactions.css

```diff
@@ -55,3 +57,10 @@
 .infonone {
   opacity: 0.25;
 }
+
+.etherscan {
+  text-align: right;
+  padding-top: 1em;
+  font-size: small;
```

Comment:
`small` is absolute. Font sizes should always be relative (e.g. `em`, `%`) for accessibility & modularity concerns.